src/Umbraco.Web.UI.Client/src/common/services/clipboard.service.js

```diff
@@ -77,7 +77,7 @@ function clipboardService($window, notificationsService, eventsService, localSto
 
     var STORAGE_KEY = "umbClipboardService";
 
-    var retriveStorage = function() {
+    var retrieveStorage = function() {
```

Comment:
I don't think we can just rename this one, it would be a breaking change for people already using the old function name. The way we typically deal with that is to leave the old name in place and redirect all usages to the new function name.

Reply:
This one should be ok - it's an internal function in the service, so won't be called directly. Point still stands though that any public function should be aliased/redirected from a new correctly spelled version.